src/SDKs/Batch/Management/Management.Batch/Microsoft.Azure.Management.Batch.csproj

```diff
@@ -2,16 +2,20 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Batch</PackageId>
-    <Description>Provides management capabilities for Azure Batch service accounts.</Description>
+    <Description>Provides management functions for the Microsoft Azure Batch services.</Description>
     <AssemblyTitle>Microsoft Azure Batch Management Library</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Management.Batch</AssemblyName>
-    <VersionPrefix>5.1.0</VersionPrefix>
-    <PackageTags>Microsoft Azure batch management</PackageTags>
+    <VersionPrefix>6.0.0</VersionPrefix>
+    <PackageTags>Microsoft Azure Batch Management windowsazureofficial</PackageTags>
     <PackageReleaseNotes>For detailed release notes, see: https://aka.ms/batch-net-management-changelog</PackageReleaseNotes>
-    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+    <NeutralLanguage>en</NeutralLanguage>
```

Comment:
@xingwu1 why are you adding all these extra properties, when they are part of AssemblyInfo.cs.
This will create an duplicate attributes error when implicit assemblyProperties setting is turned on/Off
Please model your project looking at the previous version of the project.
If you still want to add these properties, could you clarify why you need these properties in your .csproj

Reply:
@shahabhijeet I know it will conflict with AssemblyInfo.cs. The first time the extra properties add to csproj is done by you at commit 1c39c6683ff610bd4dc4f31a3a18b00e0fe71976. I am following your way. And in order to resolve the conflict, and I removed the AssemblyInfo.cs in this PR.

So I am confused, should I keep assemblyInfo.cs and removed all properties in csproj (include added by you) or the other way?